src/bin/nps.js

```
@@ -87,6 +88,11 @@ function getPSConfigFilepath() {
 }
 
 function onInit() {
+  if (getPSConfigFilepath()) {
+    if (!keyInYN(colors.yellow(`Do you want to overwrite your existing config file?`))) {
+      process.exit(1)
```

Could we use `FAIL_CODE` here instead of hard-coding `1`?